src/cli.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+/* eslint no-console:0 */
+
+import minimist from 'minimist';
+import os from 'os';
+import path from 'path';
+import sh from 'shelljs';
+import tsts from 'tsts';
+
+const argv = minimist(process.argv.slice(2));
+
+// Get a full list of spells.
+const spells = sh.ls(path.join(__dirname, 'spells'))
+  .filter(f => path.extname(f) === '.js')
+  .map(f => path.basename(f, '.js'));
+
+const help = tsts.pre`
+  Use the "cast" command to run automation sub-commands (a.k.a. "spells")
+  on a video.js plugin project using this format:
+
+    cast <spell-name> [args] [options]
+
+  Available spells are:
+
+    ${spells.join(os.EOL + '    ')}
```

Comment:
It's aligned with the indented `cast <spell-name> ...` lines. I mean, we could use `\t` there, I suppose.